pkgs/development/tools/loganalyzer/default.nix

```diff
@@ -0,0 +1,49 @@
+{ lib
+, stdenv
+, fetchurl
+, qtbase
+, qtsvg
+, wrapQtAppsHook
+}:
+
+let
+  pname = "loganalyzer";
+  appname = "LogAnalyzer";
+  version = "23.5.0";
+in
+stdenv.mkDerivation {
+  inherit pname appname version;
+
+  src = fetchurl {
```

Comment:
I didn't manage to make it build when I fetch the repo because I need to build in the `src` path and I need to fetch the submodules, so I prepared a source tar.gz file with everything I need.
Is there a way around that?

Reply:
(Thank you for the review, btw!)

Reply:
We can set `fetchSubmodule = true;` in fetchFromGitHub.

>  I need to build in the src path

It may be useful to set `sourceRoot`.

Reply:
Ah, `fetchSubmodules` does it, I found `fetchSubmodule` yesterday, and it didn't work. 😉